http-server-netty/src/main/java/io/micronaut/http/server/netty/RoutingInBoundHandler.java

```diff
@@ -425,6 +444,17 @@ private void exceptionCaughtInternal(ChannelHandlerContext ctx,
         }
     }
 
+    private List<InvocationInstrumenter> getInvocationInstrumenter() {
+        List<InvocationInstrumenter> instrumenters = new ArrayList<>(invocationInstrumenterFactories.size() + 1);
```

Comment:
If there are no instruments can we return empty or will there always be one?

Reply:
Actually, we don't need instruments here at all.